libsolidity/analysis/DocStringTagParser.cpp

```diff
@@ -47,6 +47,14 @@ bool DocStringTagParser::parseDocStrings(SourceUnit const& _sourceUnit)
 	return errorWatcher.ok();
 }
 
+bool DocStringTagParser::validateDocStringsUsingTypes(SourceUnit const& _sourceUnit)
+{
+	m_resolverRan = true;
+	auto errorWatcher = m_errorReporter.errorWatcher();
+	_sourceUnit.accept(*this);
```

Comment:
Doesn't doing this mean that we're running the whole analysis step the second time? Or was it not running on `SourceUnit` before? Even if it wasn't, it surely must have run at least on some parts of the AST.